lib/clockwork/test/rspec/matchers/have_run.rb

```diff
@@ -0,0 +1,67 @@
+module Clockwork
+  module Test
+    module RSpec
+      module Matchers
+        class HaveRun
+          def initialize(job_name = nil)
+            @job_name = job_name
+            @times_run = 1
+            @exactly = false
+          end
+
+          def matches?(clock_test)
+            if @exactly
+              clock_test.manager.times_run(@job_name) == @times_run
+            else
+              clock_test.manager.ran_job?(@job_name)
+            end
+          end
+
+          def once
+            @times_run = 1
+            @exactly = true
+
+            self
```

Comment:
I take that back, the `self` is needed for each of these.

RSpec calls `matches?` at the end of whatever operation, so, if I remove the call to `self` here, then `it { should have_run(job).once }` fails because RSpec then tries to call `matches?` on that resulting operation, which in the case of `once` is a `TrueClass`...and even if that did respond to `matches?` we want to use this class' `matches?`